app/controllers/listings_controller.rb

```diff
@@ -398,6 +398,18 @@ def update
 
     shape = get_shape(params[:listing][:listing_shape_id])
 
+    if FeatureFlagHelper.feature_enabled?(:availability) &&
+       shape.present? &&
+       shape[:availability] == :booking &&
+       @listing.availability.to_sym != :booking
+
+      bookable_res = create_bookable(@current_community.uuid_object, @listing.uuid_object, @current_user.uuid_object)
+      unless bookable_res.success
+        flash[:error] = t("listings.error.something_went_wrong_plain")
```

Comment:
Would there be a reason to use a more specific error message in this case or is it all the same to users if our BE services have connectivity issues?